track.go

```diff
@@ -57,6 +57,10 @@ type Track interface {
 	// If the error is already occured before registering, the handler will be
 	// immediately called.
 	OnEnded(func(error))
+	//Registers a handler to receive when an rtp packet is being sent
+	OnRTPWrite(func(*rtp.Packet))
+	// Register a handler to receive when an image is read
+	OnImgRead(func(image.Image, error))
```

Comment:
is this interface still conforming to the comment? 
(Reference: https://w3c.github.io/mediacapture-main/#mediastreamtrack)

Reply:
I mean it still represents the same object and nothing was removed, there is additional functionality that was added that is not part of the spec so yes it does implement that IF then some.